scenario/createsheet.go

```diff
@@ -66,6 +67,32 @@ func (settings CreateSheetSettings) Execute(sessionState *session.State,
 			return sessionState.IDMap.Add(settings.ID, genObj.GenericId, sessionState.LogEntry)
 		}
 
+		prop, err := genObj.GetPropertiesRaw(ctx)
+		if err != nil {
+			return err
+		}
+
+		// client updates two parameters, do this safely jsonparser as using a struct might "lose" values
```

Comment:
Interesting 🤔  We might create a helper for this if it becomes more commonplace to do this kind of thing, for now it's great like it is!